rails/spec/requests/home_request_spec.rb

```diff
@@ -20,5 +20,51 @@
         expect(response).to redirect_to("/users/sign_in")
       end
     end
+
+    context "orphaned stories" do
+      let(:community) { FactoryBot.create(:community, name: "Orphaned Community") }
+      let(:user) { FactoryBot.create(:user, community: community) }
+      let!(:place_1) { create(:place, community: community, region: "test place 1") }
+      let!(:place_2) { create(:place, community: community, region: "test place 2") }
+      let!(:speaker_1) { create(:speaker, community: community) }
+      let!(:speaker_2) { create(:speaker, community: community) }
+      let!(:story_1) do
+        create(
+          :story,
+          community: community,
+          places: [place_1],
+          speakers: [speaker_1],
+          desc: "This story should not be visible because it has no speaker",
+          permission_level: :anonymous
+        )
+      end
+      let!(:story_2) do
+        create(
+          :story,
+          community: community,
+          places: [place_2],
+          speakers: [speaker_2],
+          desc: "This story should not be visible because it has no place",
+          permission_level: :anonymous
+        )
+      end
+
+      def json_response
+        JSON.parse(response.body)
+      end
```

Comment:
I believe `json_response` (and `json_meta`/`json_body` are all available from `spec/support/api_helpers.rb` now. I also don't see this used in your spec, so either way, we can remove it. :)

Reply:
Ya this one was added to api_helpers after my PR was created. Removing from file